scaladoc/test/dotty/tools/scaladoc/signatures/SignatureTest.scala

```diff
@@ -110,6 +116,11 @@ abstract class SignatureTest(
 
     def processFile(path: Path): Unit = if filterFunc(path) then
       val document = Jsoup.parse(IO.read(path))
+      val documentable = document.select(".groupHeader").forEach { element =>
+        val signature = element.select(".groupHeader").eachText.asScala.mkString("")
```

Comment:
Is the .`select` here needed? From what I checked there is only one element with `groupHeader` class per extension member. So I think this can be simplified to the following

```scala
signatures += element.text
```

Also `s"$signature"` is just `signature` :smile: